src/hotspot/cpu/ppc/ppc.ad

```diff
@@ -12866,7 +12499,6 @@ instruct maxI_reg_reg_Ex(iRegIdst dst, iRegIsrc src1, iRegIsrc src2) %{
 instruct maxI_reg_reg_isel(iRegIdst dst, iRegIsrc src1, iRegIsrc src2, flagsRegCR0 cr0) %{
   match(Set dst (MaxI src1 src2));
   effect(KILL cr0);
-  predicate(VM_Version::has_isel());
```

Comment:
This makes other instruct with `match(Set dst (MaxI src1 src2));` and higher cost obsolete. `maxI_reg_reg_Ex` should be removed. There are more cases in which you have removed a predicate which makes other instruct blocks obsolete. Please check!